.github/workflows/c.yaml

```diff
@@ -34,9 +34,9 @@ jobs:
       - name: Build examples
         run: make -C c/build examples
 
-      - name: Upload coverage to codecov.io
-        run: |
-          bash <(curl -s https://codecov.io/bash) -s c/build || echo "Codecov did not collect coverage reports";
+#      - name: Upload coverage to codecov.io
+#        run: |
+#          bash <(curl -s https://codecov.io/bash) -s c/build || echo "Codecov did not collect coverage reports";
```

Comment:
Why is this commented out?

Reply:
We aren't using codecov.io anymore (we closed our account) -- so this should be migrate to SonarCloud eventually